packages/app/providers/wagmi-config.ts

```diff
@@ -1,9 +1,12 @@
+"use client";
```

Comment:
why do we need "use client" on this file?

Reply:
Because I wanted to export `config` from this file and to be able to export it from the module's index.